lab-trent/test/cowsay-test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
```

Comment:
Good start, but would love to see you start the server and test the methods/routes as well!